src/main/java/liquibase/ext/opensearch/integration/spring/LiquibaseOpenSearchAutoConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package liquibase.ext.opensearch.integration.spring;
+
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+
+@AutoConfiguration
+@EnableConfigurationProperties(SpringLiquibaseOpenSearchProperties.class)
+@ConditionalOnProperty("opensearch.uris")
+public class LiquibaseOpenSearchAutoConfiguration {
```

Comment:
I think we need to account for OpenSearch related beans (clients, etc) to be present in the context, I am happy to help you there (if it makes sense)

Reply:
`liquibase-opensearch` already pulls in the opensearch client as a dependency, so the beans will always be here; thus i don't think that we need the conditional?